packages/adapter/package.json

```diff
@@ -15,8 +15,6 @@
   "scripts": {
     "build": "ember build",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each",
```

Comment:
I removed the test commands from packages that we haven't setup testing infra in yet since we will need to update test-helpers/testem etc. still